.cirrus.yml

```diff
@@ -248,7 +248,7 @@ task:
         - ./script/tool_runner.sh build-examples --ios
       native_test_script:
         - ./script/tool_runner.sh native-test --ios --ios-destination "platform=iOS Simulator,name=iPhone 13,OS=latest"
-    - name: ios-custom_package_tests
+    - name: macOS-custom_package_tests
```

Comment:
We use all-lowercase naming for tasks, so `macos`.

LGTM with that change. Thanks!